docs/adr/0039-implement-user-tables.md

```diff
@@ -0,0 +1,108 @@
+# User Table implementation and Rollout Plan
+
+**User Story:** [EASI-3341](https://jiraent.cms.gov/browse/EASI-3341)
+
+As we look forward to future feature improvements for the EASi application, there is a need to implement User Tables. These tables will allow for per user configuration, as well as more simply storing information about each user.
+
+As this is an existing application we need to have a mechanism in place to migrate existing data that is only a username (EUAID) to reference an entity in the user table. There are also spots in the data where a user is referenced only by name, and not by username.
+
+To implement this, we need to be able to 
+1. Get information about users
+2. Store that information in the User Table
+3. Migrate existing data to use the new account reference
+
+Database migrations only happen in the DB context, so we can't query an external API when they are being run. As such, we need to come up with an approach that can update the user information at another time.
+
+## Initial Account Implementation
+
+### Considered Alternatives
+
+* Write a helper script or program that inserts user account records for all places in the DB that reference an EUAID
+* Insert place holder records
+* A combination of approaches. Using an auxiliary script, and rely on temp records for any unforeseen edge cases
+
+
+### Decision Outcome
+
+* A combination of approaches. Using an auxiliary script, and rely on temp records for any unforeseen edge cases
+
+An auxiliary script serves our current need the best. It will ensure that most users are have a record already in place before we attempt a migration. As a result, the migration should be smooth. 
+
+For cases where we cannot create a user account for an existing record, we can add placeholder account information. That will allow us to create a reference to the user table for an EUAID, and handle this edge case where user information is missing.
```

Comment:
I'd call out in this section that (at least, I think) we should attempt to approach the simpler case of mapping EUA IDs -> User Accounts, then later we can attempt to address the cases where there's user's Full Names in the DB.

Reply:
good call! Thanks Clay, I'll note that, either in this section here, and note it in the lower areas that spells out the order of implementation.

Reply:
Thanks! I implemented it